### DISCUSSION TITLE
옵셔널 체이닝 연산자란 무엇인가요?

### DISCUSSION THREAD

**Discussion by pipisebastian** [publdaze/js-study#13] [타입변환과 단축 평가]
- 옵셔널 체이닝 연산자란 무엇인가요?

**Comment by publdaze** [comment]
> - 옵셔널 체이닝 연산자란 무엇인가요?
- 객체 프로퍼티에 접근할 때 옵셔널 체이닝 연산자 좌항 값이 null 또는 undefined이면 더이상 프로퍼티를 참조하지 않고 undefined를 반환하고, 그게 아니면 계속 하위 프로퍼티 참조하는데 사용되는 연산자

**Comment by pipisebastian** [comment]
>옵셔널 체이닝 연산자란 무엇인가요?

좌항의 피연산자가 null/undefined 이라면 undefined를 반환하고, 아닐 경우 우항의 프로퍼티 참조를 이어간다
